src/App.tsx

```diff
@@ -10,31 +14,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setselectedTabId] = useState(tabs[0]);
+
+  const tabSelection = (tab: Tab) => {
+    setselectedTabId(tab);
+  };
```

Comment:
You should check if tab !== selectedTabId
<img width="589" alt="image" src="https://user-images.githubusercontent.com/99403198/220077392-38448d20-b547-45ad-ad0a-4c06e3eace9f.png">

Reply:
These tests are still failing!